ds3-autogen-c/src/main/resources/templates/request-templates/AmazonS3GetRequest.ftl

```diff
@@ -0,0 +1,29 @@
+<#-- ********************************** -->
+<#-- Generate "AmazonS3 Get Request" -->
+<#--   Input: Source object             -->
+<#-- ********************************** -->
+<#list getRequests() as requestEntry>
+    <#if (requestEntry.getClassification().toString() == "spectras3") && (requestEntry.getVerb().toString() == "GET")>
```

Comment:
That's for the path, but that logic seems like it should be in the generator and not in the code generator.